### ISSUE TITLE
Show least CR position on exchange

### ISSUE THREAD

**Issue by froooze** [bitshares/bitshares-ui#3042]
**Is your feature request related to a problem? Please describe.**
A margin trader is interested, what CR the lowest CR of all margin position is.

**Describe the solution you'd like**
Show the lowest margin position in the tooltip of the top-bar Collateral Ratio 

**Describe alternatives you've considered**
None

**Comment by startailcoon** [comment]
This is already present, but only when a market price is below the least CR position.

![image](https://user-images.githubusercontent.com/12114550/63376981-524c9100-c38f-11e9-8588-46111aa88485.png)

**Comment by froooze** [comment]
@startailcoon: 
You reference to the price until margin calls happens, I want to have the CR from the lowest margin position.

**Comment by startailcoon** [comment]
> 
> 
> @startailcoon:
> You reference the price until a margin calls happens, I want to have the CR from the lowest margin position.

No, the values at the "call limit" is the least collateral position as seen on the margin position list.

![image](https://user-images.githubusercontent.com/12114550/64081470-c842e780-cd01-11e9-8f2b-29f91c13b511.png)

![image](https://user-images.githubusercontent.com/12114550/64081476-dee93e80-cd01-11e9-9566-409d6491cad9.png)

**Comment by froooze** [comment]
> No, the values at the "call limit" is the least collateral position as seen on the margin position list.

The UI doesn't show the least CR (1.330) on the Exchange!
